tck/src/main/java/org/eclipse/microprofile/lra/tck/TckInvalidSignaturesTests.java

```diff
@@ -64,7 +66,7 @@
 
     @ArquillianResource
     private Deployer deployer;
-    
```

Comment:
I am not sure if the community has a preference decided here yet but I wonder if formatting of all the sources could be done at the same time and maybe add a checkstyle rule to the build?

Reply:
Our maven-checkstyle-plugin rules are [here](https://github.com/eclipse/microprofile-lra/blob/master/pom.xml#L204). When I created the rules I copied them from another MP pom (and they all seemed to be using the same set but I haven't checked since). We don't have any rules around blank lines though (but I think it is doable with a regex).

Reply:
This was a forgotten switch on my side. It's a manual revert so if it's required I will revert it.

Reply:
Although I don't think blank lines were effectively added, lines that would be blank other than whitespace were modified. I think it looks like trailing whitespace was removed? If you look at https://github.com/eclipse/microprofile-lra/pull/273/files you will see what I mean.

As the changes meets the current checkstyle rules of the project then that would suggest (to me, non-committer on this project) it does not need reverting.  I would comment though that even if it is not in the checkstyle rules, it seems MP Config has a .editorconfig for non-.md and non-.adoc files: https://github.com/eclipse/microprofile-config/blob/828ee2fee4ccbef6701943bb96d7be187408fd63/.editorconfig#L12 which I assume might remove the trailing whitespace in at least the .java files in editors that understand the format (just surmising here though as I am not an expert on the .editorconfig file type)

I also haven't tried the following, but I do wonder if preventing trailing whitespace is something the community want then https://checkstyle.org/config_whitespace.html#NoWhitespaceAfter might detect this?

Reply:
Actually, I can't be sure if it does or doesn't meet the checkstyle rules as I don't know if CI is ran on this PR and if so whether it tests the checkstyle?

Reply:
CI was not ran on the PR

Reply:
We do not run CI on PRs.

The NoWhitespaceAfter rule would be useful.

I would suggest we do not apply the same standard of rigour to pre GA PR's when it comes to formatting, after all the GA release should be clean from that point of view and thereafter we can apply the higher bar for PRs with respect to incidental changes such as cleaning up white space.